From 163df4d736aae19e8054cf977150b2791615913e Mon Sep 17 00:00:00 2001
From: fli <fupan.li@windriver.com>
Date: Fri, 21 Jul 2017 01:14:50 -0700
Subject: [PATCH] rpm: disable the debug to ignore the tons of error info

To ignore tons of the following error infos:

=> success
error: db3cpget:/buildarea1/mbp/tmp/work/core2-64-wrs-linux/rpm/5.4.14-r0/rpm-5.4.14/rpmdb/db3.c:1502: dbcursor->pget(-30999): BDB0063 DB_BUFFER_SMALL: User memory too small for return value
==> retry (1) db3cpget (4648)
error: db3cpget:/buildarea1/mbp/tmp/work/core2-64-wrs-linux/rpm/5.4.14-r0/rpm-5.4.14/rpmdb/db3.c:1502: dbcursor->pget(-30999): BDB0063 DB_BUFFER_SMALL: User memory too small for return value
==> retry (2) db3cpget (4868)

Signed-off-by: fli <fupan.li@windriver.com>
---
 rpmdb/db3.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpmdb/db3.c b/rpmdb/db3.c
index 0043967..82c5c0a 100644
--- a/rpmdb/db3.c
+++ b/rpmdb/db3.c
@@ -4,7 +4,7 @@
  */
 
 /*@unchecked@*/
-static int _debug = 1;	/* XXX if < 0 debugging, > 0 unusual error returns */
+static int _debug = 0;	/* XXX if < 0 debugging, > 0 unusual error returns */
 
 #include "system.h"
 
-- 
2.11.0

